jenkins-pipelines/longevity-200gb-48h-asymmetric.jenkinsfile

```diff
@@ -0,0 +1,12 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    region: 'eu-west-1',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: 'test-cases/longevity/longevity-200GB-48h-verifier-LimitedMonkey-tls-asymmetric.yaml'
```

Comment:
As mentioned in the comment below, here you just need to have the original test_config + "configurations/db_nodes_shards_selection.yaml".

Reply:
Done